lib/patterns.coffee

```diff
@@ -83,11 +84,20 @@ exports.login = (callback) ->
 #		console.log(token)
 ###
 exports.loginWithToken = (callback) ->
-	form.ask
-		label: 'What\'s your token? (visible in the preferences page)'
-		name: 'token'
-		type: 'text'
-	, callback
+	###
+	Use of prompt package instead of inquirer in order to
```

Comment:
Very nice comment :) Just for aesthetics purposes, can you add an empty line between the start of the command and the function signature (it makes it easier to the eye)?